qea7-Metal/src/test/java/pageObjects/HomePage.java

```diff
@@ -0,0 +1,80 @@
+package pageObjects;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import foundation.PageObjectBase;
+
+public class HomePage extends PageObjectBase {
+
+	private static final String URL = "http://invenauto.tech/index.php";
+	private WebDriver driver;
+
+	public HomePage(WebDriver driver) {
+		this.driver = driver;
+	}
+
+	public HomePage navigate() {
+		driver.navigate().to(URL);
+		return this;
+	}
+
+	public List<WebElement> getAllCategories() {
+		List<WebElement> categoryNames = this.driver
+				.findElements(By.xpath("//ul[@class='sf-menu clearfix menu-content sf-js-enabled sf-arrows']/li"));
+
+		return categoryNames;
+	}
+
+	public HomePage clickToEachCategory() {
+		List<WebElement> categories = this.driver
+				.findElements(By.xpath("//ul[@class='sf-menu clearfix menu-content sf-js-enabled sf-arrows']/li"));
+
+		for (int i = 0; i < categories.size(); i++) {
+			categories.get(i).click();
+
+			this.driver.navigate().back();
```

Comment:
Does clicking the category cause a navigate to the next page? That's why we are navigating back to the homepage?
You might want to rethink this approach. Going back and forth between pages in a single method might be brittle in the long run.